src/features/companyOverview/index.tsx

```diff
@@ -40,42 +40,31 @@ const Link = styled.a`
 const Description = styled.div`
     font-size: 16px;
     max-height: 140px;
-`
-
-const OverflowContainer = styled.div`
     overflow: auto;
-    max-height: 200px;
+    max-height: 100px;
 `
 
-
-export const Company: FC<CompanyOverview> = ({ companyName, symbol, website, description }) => {
-    return (
-        <>
-            <Name>{companyName} ({symbol})</Name>
-            <Website>
-                {website ?
-                <Link href={website}><i>{website}</i></Link>
-                :
-                <span><i>{website}</i></span>
-            }
-            </Website>
-            <OverflowContainer>
-                <Description>{description}</Description>
-            </OverflowContainer>
-        </>
-    )
-}
-
 export const CompanyOverviewComponent: FC<{}> = () => {
 
-    const { ...companyProps }: CompanyOverview = useSelector((store: AppState) => store.companyOverview)
+    const { symbol, companyName, website, description }: CompanyOverview = useSelector((store: AppState) => store.companyOverview)
 
     return (
         <CompanyOverviewContainer>
             <Title>COMPANY OVERVIEW</Title>
-            {companyProps.symbol
-                ? <Company {...companyProps} />
-                : <Loader className='margin-top: 50px; margin-bottom: 50px;' size={50} seperation={2} speed={1.4} />
+            {
+                !symbol
+                ? <Loader className='margin-top: 50px; margin-bottom: 50px;' size={50} seperation={2} speed={1.4} />
```

Comment:
maybe better to move CSS into a different file

Reply:
this is a method to extend the styling of a styled component and not your standard CSS classname.

Reply:
maybe you can tell me about it later on